pylsqpack 0.3.23 with v1 recipe, abi3, arches, maintainer

bollwyvl opened the pull request:
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

References:
- includes #10
- would obviate the need for #9 (and future migrations)
- unblocks https://github.com/conda-forge/aioquic-feedstock/issues/5

Changes:
- [x] use `rattler-build`
- [x] use `pixi`
- [x] use `abi3`
- [x] add more arches 
- [x] add tests with coverage
- [x] add link for docs
- [x] add self as maintainer :wave:

regro-cf-autotick-bot committed:
updated v0.3.23

regro-cf-autotick-bot committed:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.10.10.13.43.2

bollwyvl committed:
v1 recipe, abi3, maintainer

bollwyvl committed:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.10.10.13.43.27

Other tools:
- conda-build 25.9.0
- rattler-build 0.48.1
- rattler-build-conda-compat 1.4.5

bollwyvl committed:
more test, tidying

bollwyvl committed:
add docs url

bollwyvl committed:
fix tests

bollwyvl committed:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.10.10.13.43.27

Other tools:
- conda-build 25.9.0
- rattler-build 0.48.1
- rattler-build-conda-compat 1.4.5

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

bollwyvl commented:
@conda-forge/pylsqpack ready for review

bollwyvl committed:
start merge of main

bollwyvl committed:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.10.13.17.18.10

Other tools:
- conda-build 25.9.0
- rattler-build 0.48.1
- rattler-build-conda-compat 1.4.5

bollwyvl commented:
@conda-forge-admin please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to rerender for you, but it looks like there was nothing to do.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/18480857288](https://github.com/conda-forge/conda-forge-webservices/actions/runs/18480857288). Examine the logs at this URL for more detail.</sub>

bollwyvl commented:
@conda-forge-admin please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to rerender for you, but it looks like there was nothing to do.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/18668877310](https://github.com/conda-forge/conda-forge-webservices/actions/runs/18668877310). Examine the logs at this URL for more detail.</sub>

bollwyvl commented:
@conda-forge/core this PR, which would resolve all open PRs and add a maintainer, has been open for a week without any comment from @conda-forge/pylsqpack. A review would be lovely!

h-vetinari reviewed (approved):
Cool! :)

bollwyvl commented:
Thank you!
